actions/get_issue.py

```diff
@@ -9,12 +9,26 @@
 class GetJiraIssueAction(BaseJiraAction):
     def run(self, issue_key, include_comments=False, include_attachments=False,
             include_customfields=False, include_components=False, include_subtasks=False,
-            include_links=False):
+            include_links=False, sanitize_formatting=False):
         issue = self._client.issue(issue_key)
         result = to_issue_dict(issue=issue, include_comments=include_comments,
                                include_attachments=include_attachments,
                                include_customfields=include_customfields,
                                include_components=include_components,
                                include_subtasks=include_subtasks,
                                include_links=include_links)
-        return result
+
+        def strip_braces(data):
+            if isinstance(data, dict):
+                return {k: strip_braces(v) for k, v in data.items()}
+            elif isinstance(data, list):
+                return [strip_braces(element) for element in data]
+            elif isinstance(data, str):
+                return data.replace("{{", "").replace("}}", "")
+            else:
+                return data
+
+        if sanitize_formatting:
```

Comment:
This could be condensed to `return strip_braces(result) if sanitize_formatting else result`

Reply:
Condensed in lastest commit.